README.md

```diff
@@ -634,7 +634,7 @@ $ just --list --list-heading ''
 
 ### Aliases
 
-Aliases allow recipes to be invoked with alternative names:
+Aliases allow using alternative names for recipes when invoking them at the command line:
```

Comment:
How about this:

```suggestion
Aliases allow recipes to be invoked on the command line with alternative names:
```

Reply:
I'm not sure! For this change, I think the important thing is to change `be invoked` to `be invokeed on the command line`, and I don't mind the previous wording.